cppcache/integration-test/ThinClientPdxSerializers.hpp

```diff
@@ -35,41 +37,41 @@ using PdxTests::TestPdxSerializerForV2;
 using PdxTests::V1CLASSNAME2;
 using PdxTests::V2CLASSNAME4;
 
-static const char *CLASSNAME1 = "PdxTests.PdxType";
-static const char *CLASSNAME2 = "PdxTests.Address";
+static const char* CLASSNAME1 = "PdxTests.PdxType";
```

Comment:
The default in CLion is to put the `*` on the variable rather than the type.  I think CLion supports integrated Clang but haven't looked in a while.  See [this](https://stackoverflow.com/questions/45656445/clion-formatting-of-pointer-references) for changing those settings.

Reply:
CLion is wrong.

Reply:
CLion lacks the ability to actually apply the Google rule here. They flipped a coin and chose right alignment. We mostly have left alignment. The google rule is either but be consistent. Clang-format will determine the majority in the file and correct the minority.